java/src/test/android/.gitignore

```diff
@@ -0,0 +1,90 @@
+# Built application files
```

Comment:
are these for files generated by a build? do those get put in java/src/test/android or a build directory?

Reply:
In general these will go to build dir in our pipeline, however, you can run gradle under java/src/test/android as well, which will create many files as changed file, adding a gitigore will filter out all the files